### PR TITLE
added client perf tests job

### PR DESCRIPTION
**Pull request by rajatbansal0905**
Added Client Perf Tests job in the build Workflow as a stage. Ideally should not block the current runs.

### CODE REVIEW

File: .github/workflows/hotstar-android-build-workflow.yml [hunk 1]
```diff
@@ -86,6 +86,25 @@ jobs:
...
+          repository: pe-infra-scripts
+          path: .
+          ref: perf-harness
+
+      - name: Prerequisites
+        run: pip3 install requests
+
+      - name: Upload and install build on Mozark devices
+        working-directory: pe-infra-scripts/perf-lab
+        run: python3 mozarkTest.py mozarkEnvVariablesAndroidX.json MEA Test
```
Comment (ajaychoudhary-hotstar) [inline .github/workflows/hotstar-android-build-workflow.yml:106]: why MEA is hardcoded? it should be IN for India?

Comment (rajatbansal0905) [reply .github/workflows/hotstar-android-build-workflow.yml:106]: this is only for my testing my script in Github Actions. will add another step, once this done

**Comment by rajatbansal0905** [comment]
successful run - https://github.com/hotstar/hotstar-android-mobile/runs/7423275816?check_suite_focus=true

**Review by raj-dhamsaniya — APPROVED** [review]

**Comment by rajatbansal0905** [comment]
successful run for hotstar-android-livingroom -> https://github.com/hotstar/hotstar-android-livingroom/runs/7430517627?check_suite_focus=true

**Review by ajaychoudhary-hotstar — APPROVED** [review]

**Pull request merged**